src/containers/Bookings.js

```diff
@@ -2,22 +2,37 @@
 import React, { Component } from "react";
 import Search from "../components/search/Search.js";
 import Results from "../components/results/Results.js";
-import * as FakeBookings from "../data/fakeBookings.json";
+
+// import * as FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      FakeBookings: '',
-      term: '',
+      FakeBookings: [],
+      term: null,
       result: false,
       direction: {
-        sortin: 'asc'
+        sortin: 'asc',
+        error: null,
       },
-      text: ''
+      text: null,
```

Comment:
I think you're missing a loading state? When you start your HTTP fetch, you're not showing anything to the user. So they might be confused about what is happening (and might click the button lots of times)